### PR TITLE
Fixes missing items for Eaton M2 management card

### PR DESCRIPTION
**Pull request by ajsiersema**
System overview items like Hardware, Version and Serial and sensors like Temperature and Humidity sensors where missing for UPSes equipped with the Eaton M2 management card.
The M2 is RFC 1628 compatible and doesn't support MG-SNMP-UPS-MIB/upsmg.
Use XUPS-MIBS instead.

Tested on the:
9PX 5000i
9PX 6000i
5P 1150
5PX 1500
(v1.7.5)

Also see:
https://community.librenms.org/t/force-sw-hw-information-from-another-mib/11417

Please review.
I'd lie to add "rfc1628_compat: true"., which should reduce eatonups specific code, and also provide items like "Runtime" (Runtime remaining) and "Load" (device_load). Adding it together with the current eatonups code will make some items appear twice (e.g. Frequency/Bypass).

If we can rely on sysObjectID's .1.3.6.1.4.1.534.1|2 to _only_ match M2s, the rfc1628 code could replace some (most?) eatonups specific code, like e.g. includes/discovery/sensors/charge/eatonups.inc.php
(maybe the rfc1628 code was added after the eatonups code ?)

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/eatonups.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+mib: EATON-OIDS:EATON-EMP-MIB:XUPS-MIB
```
Comment (murrant) [inline includes/definitions/discovery/eatonups.yaml:1]: only the XUPS-MIB is used here.

File: includes/definitions/eatonups.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ over:
     - { graph: device_frequency, text: Frequencies }
 mib_dir:
     - ups
+    - mge
```
Comment (murrant) [inline includes/definitions/eatonups.yaml:11]: Why did you add the mge mib directory, that is used in a different OS.

File: includes/polling/os/eatonups.inc.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+$hardware = trim(snmp_get($device, 'upsIdentModel.0', '-OQv', 'UPS-MIB'), '" ');
```
Comment (murrant) [inline includes/polling/os/eatonups.inc.php:3]: trim() is not needed, this should code should be implemented in the new style.  See LibreNMS\OS\Qnap.php

**Comment by dagbdagb** [comment]
See #11609 for where I got sidetracked. Plus an explanation for the madness branded as Eaton.

**Comment by ajsiersema** [comment]
> See #11609 for where I got sidetracked. Plus an explanation for the madness branded as Eaton.

I honestly searched the git repo first, I can't imagine how I have missed this one.

I think we're both trying to achieve the same, can we merge our efforts, how can I help ?

**Comment by Jellyfrog** [comment]
Thank you for your contribution but we're closing this due to inactivity.  
Feel free to reopen if you plan on finishing it!
